examples/tutorial 1c - GreedyMIS Example Use Case.ipynb

```diff
@@ -0,0 +1,313 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d276a71c",
+   "metadata": {},
+   "source": [
+    "# Using Greedy MIS Solver\n",
+    "\n",
+    "This notebook demonstrates how to solve the Maximum Independent Set (MIS) problem using the GreedyMIS algorithm provided by the `MIS` library. In this notebook, we present different solving modes:\n",
+    "\n",
+    "- Classical solver \n",
+    "- Classical solver with greedy decomposition\n",
+    "- Quantum solver\n",
+    "- Quantum solver with greedy decomposition"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fb1d6d08",
+   "metadata": {},
+   "source": [
+    "# Introduction to GreedyMIS\n",
+    "The GreedyMIS algorithm is a heuristic strategy for solving the Maximum Independent Set (MIS) problem by decomposing a graph into smaller subgraphs and solving each part either classically or using a QPU. This approach is particularly valuable when working with hardware-constrained systems such as Rydberg atom arrays, where only specific subgraphs (e.g., unit-disk graphs) can be embedded and solved natively.\n",
```

Comment:
```suggestion
    "The GreedyMIS algorithm is a heuristic strategy for solving the Maximum Independent Set (MIS) problem by decomposing a graph into smaller subgraphs and solving each part either classically or using a QPU.\n",
    "This approach is particularly valuable for two reasons:
    - it supports parallelization, if you have access to several quantum devices or to a single device that can perform several distinct computations concurrently;
    - it can be used to tailor the graph to hardware-constrained systems such as Rydberg atom arrays, where only specific subgraphs (e.g., unit-disk graphs) can be embedded and solved natively.\n",
```